ddp-workspace/projects/ddp-testboston/src/assets/i18n/en.json

```
@@ -89,7 +89,7 @@
         {
           "Question": "Can anybody join this study?",
           "Paragraphs": [
-            "TestBoston is open to a randomly selected group of adult residents from the greater Boston area, who have received care at Brigham and Women’s Hospital at least once in the past year. Each randomly selected patient will receive an invitation to the study by postal mail or email."
+            "Yes. TestBoston is open to all adults in the greater Boston area. To volunteer, please fill in a brief screening survey here. If you volunteer and are selected to participate, you will be emailed an invitation code to register."
```

> To volunteer, please fill in a brief screening survey **here**.

I think it would be better to make "here" pointing to https://rally.partners.org/study/testboston, what do you think?
```suggestion
            "Yes. TestBoston is open to all adults in the greater Boston area. To volunteer, please fill in a brief screening survey <a class=\"Link Link_semi-bold\" href=\"https://rally.partners.org/study/testboston\" target=\"_blank\">here</a>. If you volunteer and are selected to participate, you will be emailed an invitation code to register."
```